--- helper_scripts/build_python.sh.orig	2016-07-05 11:03:49.503503611 -0500
+++ helper_scripts/build_python.sh	2016-07-05 11:26:36.431063437 -0500
@@ -33,7 +33,7 @@
     fi
 fi
 
-./configure $EXFLAGS --prefix="$PY_HOME" >> "$INSTALL_LOG" 2>&1
+./configure $EXFLAGS --prefix="$PY_HOME" "LDFLAGS=-L/usr/local/openssl/lib -L/usr/local/openldap2.4/lib -Xlinker -rpath=/usr/local/openssl/lib -Xlinker -rpath=/usr/local/openldap2.4/lib" "CPPFLAGS=-I/usr/local/openssl/include -I/usr/local/openldap2.4/include" >> "$INSTALL_LOG" 2>&1
 if [ $? -gt 0 ]; then
     eval "echo \"$INSTALLING_PYTHON\""
     seelog
--- helper_scripts/main_install_script.sh.orig  2016-07-06 15:54:01.017090168 -0400
+++ helper_scripts/main_install_script.sh   2016-07-06 15:56:09.385829994 -0400
@@ -434,7 +434,7 @@
     if [ -f ./buildenv.sh ]; then
         rm -f ./buildenv.sh
     fi
-    sh ./preflight -q
+    sh ./preflight -q "LDFLAGS=-L/usr/local/openssl/lib -L/usr/local/openldap2.4/lib -Xlinker -rpath=/usr/local/openssl/lib -Xlinker -rpath=/usr/local/openldap2.4/lib" "CPPFLAGS=-I/usr/local/openssl/include -I/usr/local/openldap2.4/include"
     if [ $? -gt 0 ] || [ ! -f "buildenv.sh" ]; then
         echo "$PREFLIGHT_FAILED_MSG"
         exit 1
